__tests__/app.test.js

```diff
@@ -66,3 +66,25 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("should respond with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.article.forEach((element) => {
```

Comment:
Can we please add an assertion for checking how many articles we get back (i.e. prove that we have them all?)